.github/actions/backup-paas-database/action.yml

```diff
@@ -0,0 +1,46 @@
+name: Backup DB
+description: Backup GOV.UK PaaS DB upload GitHub artefact
+
+inputs:
+  environment:
+    description: The name of the environment
+    required: true
+  staging-username:
+    description: PAAS staging username
+    required: false
+    default: ""
+  staging-password:
+    description: PAAS staging password
+    required: false
+    default: ""
+
+runs:
+  using: composite
+
+  steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+
+    - name: Setup cf cli (staging)
+      if: inputs.environment == 'staging'
+      uses: DFE-Digital/github-actions/setup-cf-cli@master
+      with:
+        CF_USERNAME: ${{ inputs.staging-username }}
+        CF_PASSWORD: ${{ inputs.staging-password }}
+        CF_SPACE_NAME: early-careers-framework-staging
+        INSTALL_CONDUIT: true
+
+    - name: Setup postgres client
+      uses: DFE-Digital/github-actions/install-postgres-client@master
+
+    - name: Backup database
+      shell: bash
+      run: |
+        cf conduit ecf-postgres-${{ inputs.environment }} -- pg_dump -E utf8 --clean --compress=1 --if-exists --no-owner --no-privileges --verbose -f backup-${{ inputs.environment }}.sql.gz
```

Comment:
db is npq-registration-${{ inputs.environment }}-db

Reply:
thank you, will fix. Was pulled onto something else and left this in flight 😩